src/Components/Files/File Operation/select.ts

```diff
@@ -108,15 +94,9 @@ const Initializer = () => {
 				selectFirstFile();
 				return;
 			}
-			if (
-				Array.from(latestSelected.parentNode.children).indexOf(
-					latestShiftSelected
-				) <
-				Array.from(latestSelected.parentNode.children).indexOf(
-					latestSelected
-				)
-			)
+			if (childIndex(latestSelected.parentNode) < childIndex(latestSelected.parentNode)) {
```

Comment:
I figured out why this broke the function, 

let a = `childIndex(latestSelected.parentNode)`, then
`if(a < a)` should always return false

Reply:
Good catch! I did not see this issue :(

Reply:
Haha, I wasted almost an hour to figure it out lmao, thank you so much tho.

If you got time, please help me refactor my code haha (some of them are so ugly, but not contextmenu cuz I'll surely refactor whole of it), or maybe help contribute by picking any issue there 

TYSM

Reply:
Will do my best!